src/components/navbar/index.cjsx

```diff
@@ -93,6 +94,7 @@ module.exports = React.createClass
             title={<UserName/>}
             ref='navDropDown'>
             {menuItems}
+            <AdminConsole/>
```

Comment:
I can move what I have in `src/components/navbar/admin-console.cjsx` into `src/components/navbar/username.cjsx`.  I'm not sure how else I can do it...